analysis/src/analysis.js

```diff
@@ -48,15 +49,18 @@ class Analysis {
       var versionOrSha = attributes.sha ? attributes.sha : attributes.version;
       this.bower.prune().then(() => {
         return this.bower.install(attributes.owner, attributes.repo, versionOrSha);
-      }).then(result => {
+      }).then(async (result) => {
         if (!fs.existsSync(result.root)) {
           Ana.fail("analysis/processNextTask", taskAsString, "Installed package not found");
           reject({retry: false, error: Error("Installed package not found")});
           return;
         }
 
+        // Move /bower_components/element/* to /bower_components/*.
```

Comment:
shouldn't this be "Move /bower_components/element* to /*"?

Reply:
Yes! Thanks